src/loaders/jobs/taskfarmer/task_generator.py

```diff
@@ -49,11 +49,12 @@
 256 cores.
 '''
 
-TOOLS_AVAILABLE = ['gtdb_tk', 'checkm2']
+TOOLS_AVAILABLE = ['gtdb_tk', 'checkm2', 'microtrait']
 
 # estimated execution time (in minutes) for each tool to process a chunk of data
 TASK_META = {'checkm2': {'chunk_size': 5000, 'exe_time': 60},
-             'gtdb_tk': {'chunk_size': 1000, 'exe_time': 65}}
+             'gtdb_tk': {'chunk_size': 1000, 'exe_time': 65},
+             'microtrait': {'chunk_size': 5000, 'exe_time': 60}}
```

Comment:
is this a guess or does it actually run 5000 in 60m?

Reply:
it's an estimation. It takes about 2 min per genome. (60/2) * 200 = ~6000 genomes per hour if we utilize 200 cores.

Reply:
ok, if we start seeing timeouts we can drop the chunk size or increase the time I guess